src/test/java/uk/gov/dhsc/htbhf/page/component/RadioButton.java

```diff
@@ -11,6 +11,8 @@
 
     public static final String YES = "yes";
     public static final String NO = "no";
+    public static final String TEXT = "text";
```

Comment:
I would have said these constants sat better on `SendCodePage` as I don't think they'd be used anywhere else?

Reply:
Yes, you're probably right, they're probably more to do with Send Code than the more generic RadioButton. I'll change in the next PR so I don't need re-approvals.